src/components/Card/LaptopCard.tsx

```diff
@@ -1,75 +1,107 @@
-import { laptopData } from './constants';
+import { FC } from 'react';
+import { Rate } from 'antd';
 import styles from './card.module.css';
-import { Card, Rate } from 'antd';
-import { useProductCardHandlers } from '@/hooks/hooks';
+import { rateImg, rateEmptyImg } from '@/assets/constants';
+import { Link } from 'react-router-dom';
+import { useProductCardHandlers } from '@/hooks/useProductCardHandlers';
+import { IAccessory } from '@/interfaces/interfaces';
 import { BasketIcon } from '@/assets/icons/BasketIcon';
-import { rateEmptyImg, rateImg } from '@/assets/constants';
-import { ICard } from '@/interfaces/interfaces';
-import { FC } from 'react';
+import classNames from 'classnames';
 
-interface ILaptopCardProps {
-  product: ICard;
+interface ILaptopCard {
+  product: IAccessory;
+  classname: string;
 }
 
-export const LaptopCard: FC<ILaptopCardProps> = ({ product }) => {
+export const LaptopCard: FC<ILaptopCard> = ({ product, classname }) => {
   const { isLiked, handleClickBuy, handleClickLike } = useProductCardHandlers();
 
   const productRating = product.rate ?? 0;
+  const encodedTitle = encodeURIComponent(product?.title || '');
 
   return (
-    <div>
-      {laptopData.map((item) => (
-        <Card className={styles.cardConatiner} key={item.id}>
-          <div className={styles.cardContainerTop}>
+    <>
+      <Link
```

Comment:
This component takes a product as a prop, which implements the IAccessory interface. The product will always have an id parameter, so we can avoid using optional chaining to access it.